test/csuite/wt5212_incremental_backup/main.c

```diff
@@ -0,0 +1,779 @@
+     */
+    testutil_check(session->create(session, uri4, "key_format=i,value_format=S"));
+
+    add_bulk_load(session, 0, 0, uri4);
+    // ID 15 is used for logged table
+    take_full_backup(session, ITERATIONS_MULTIPLIER * 3);
+    take_incr_backup(session, ITERATIONS_MULTIPLIER * 3);
+    printf("Dumping and comparing data\n");
+    testutil_check(compare_backups(ITERATIONS_MULTIPLIER * 3, uri4));
+
+    /*
+     * Insert bulk data into not logged table
+     */
+    testutil_check(
+      session->create(session, uri5, "key_format=i,value_format=S,log=(enabled=false)"));
+
+    add_bulk_load(session, 0, 0, uri5);
+    // ID 16 is used for not logged table
+    take_full_backup(session, 16);
+    take_incr_backup(session, 16);
+    printf("Dumping and comparing data\n");
+    testutil_check(compare_backups(16, uri5));
+
+    new_object = false;
+}
+
+int
+main(int argc, char *argv[])
+{
+    struct stat sb;
+    WT_CONNECTION *wt_conn;
+    WT_CURSOR *backup_cur;
+    WT_SESSION *session;
+    int i, ret;
+    char cmd_buf[256];
+
+    (void)argc; /* Unused variable */
+    (void)testutil_set_progname(argv);
+
+    testutil_check(
+      __wt_snprintf(cmd_buf, sizeof(cmd_buf), "rm -rf %s && mkdir -p %s/%s", home, home, logpath));
+    testutil_check(system(cmd_buf));
+    testutil_check(wiredtiger_open(home, NULL, CONN_CONFIG, &wt_conn));
+
+    setup_directories();
+    testutil_check(wt_conn->open_session(wt_conn, NULL, NULL, &session));
+    testutil_check(session->create(session, uri, "key_format=S,value_format=S"));
+    testutil_check(session->create(session, uri2, "key_format=S,value_format=S"));
+
+    printf("*** Add data, checkpoint, take backups and validate ***\n");
+    add_data_validate_backups(session);
+
+    printf("*** Remove old records and validate ***\n");
+    remove_all_records_validate(session);
+
+    printf("*** Drop old and add new objects ***\n");
+    drop_old_add_new_objects(wt_conn);
+
+    printf("*** Create previously dropped object and add new content ***\n");
+    create_dropped_object_add_new_content(wt_conn);
+
+    printf("*** Insert data into Logged and Not-Logged tables ***\n");
+    insert_bulk_data(wt_conn);
+
+    printf("Close and reopen the connection\n");
+    /*
+     * Close and reopen the connection to illustrate the durability of id information.
+     */
+    testutil_check(wt_conn->close(wt_conn, NULL));
+    testutil_check(wiredtiger_open(home, NULL, CONN_CONFIG, &wt_conn));
+    testutil_check(wt_conn->open_session(wt_conn, NULL, NULL, &session));
+    /*
+     * We should have an entry for MAX_ITERATIONS-1 and MAX_ITERATIONS-2. Use the older one.
+     */
+    testutil_check(__wt_snprintf(cmd_buf, sizeof(cmd_buf), "incremental=(src_id=ID%d,this_id=ID%d)",
+      MAX_ITERATIONS - 2, MAX_ITERATIONS));
+    testutil_check(session->open_cursor(session, "backup:", NULL, cmd_buf, &backup_cur));
+    testutil_check(backup_cur->close(backup_cur));
+
+    /*
```

Comment:
Everything that's doing backup from here to the end should not be needed.

Reply:
Fixed